source/docs/concepts/front-commerce-folder-structure.md

```diff
@@ -24,7 +24,7 @@ src/
 
 ## src/config/
 
-It contains all the files that allow you to configure the behavior of Front-Commerce.
+The config contains all the files that allow you to configure the behaviour of Front-Commerce.
```

Comment:
I don't know if it's written somewhere (or maybe I'm even wrong), but I thought we usually write _American English_ (again not a big deal :))